.travis.yml

```diff
@@ -13,5 +13,5 @@ script:
   - yarn test
   - pip install .
   - jupyter labextension install .
-  - python -m jupyterlab.browser_check
```

Comment:
Should we be removing this check? For e.g., if JupyterLab adds more tests to that package, we won't inherit them.

Reply:
Added it back